pendulum/date.py

```diff
@@ -137,11 +137,11 @@ def day_of_year(self):
 
         :rtype: int
         """
-        k = 1 if self.is_leap_year else 2
+        k = 1 if self.is_leap_year() else 2
 
         return (
             (275 * self.month) // 9
-            - k * (self.month + 9) // 12
+            - k * ((self.month + 9) // 12)
```

Comment:
I don't think it's the same result either way:

```python
>>> k = 12
>>> mon = 1
>>> k * ((mon + 9) // 12)
0
>>> k * (mon + 9) // 12
10
```

It may be that it's the same for all values k can currently hold I suppose, but if the version with parens is what you're expecting, it's probably cleaner to include the parens so that the semantics are better understood.